bazel/test_secondary_lto_cache/MODULE.bazel.lock

```diff
@@ -0,0 +1,3710 @@
+{
```

Comment:
Does it make sense to check in these large lock files into source control given that they are just tests?  I guess so?

I wonder if were should move these test directories under the top level `test/` (not has part of this PR).

Reply:
Lets do that as a followup then.  Thanks.